lnrpc/verrpc/verrpc.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package verrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/verrpc";
+
+message VersionRequest {
+};
+
+message Version {
+    /// A verbose description of the daemon's commit.
+    string commit = 1;
+
+    /// The SHA1 commit hash of the daemon.
+    string commit_hash = 2;
+
+    /// The semantic version.
+    string version = 3;
+
+    /// The major application version.
+    uint32 app_major = 4;
+
+    /// The minor application version.
+    uint32 app_minor = 5;
```

Comment:
We could also consider a dedicated version number for the API. Only increment that with changes. Downside is extra work, don't know about upside.

Reply:
i think the `app_major` is likely good enough for that, since we try not to introduce breaking changes in point releases